.travis.yml

```diff
@@ -5,10 +5,19 @@
 
 install:
   - cd ..
-  - git clone https://github.com/google/closure-compiler.git
+  - git clone --depth 1 https://github.com/google/closure-compiler.git
```

Comment:
I put together a separate PR for this. good idea:
https://github.com/google/closure-library/pull/538